content/shmem_test_nbe.tex

```diff
@@ -0,0 +1,38 @@
+\apisummary{
+    The request test routine provides a method for querying the status
+    of the request.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void shmem_test_nbe(shmem_request_t *req, int *status);
+\end{Csynopsis}
+
+\begin{apiarguments}
+    \apiargument{IN}{req}{Handle to the request object.}
+    \apiargument{OUT}{status}{The status of the request.}
+\end{apiarguments}
+
+\apidescription{
+    The routine returns one via the status argument if the request is
+    finished, otherwise zero is returned.
```

Comment:
Suggest: "one" &#8594; 1 and "zero" &#8594; 0 for consistency with `shmem_test`

Reply:
Does "finished" imply local completion (i.e., the local buffer can be [re]used) or remote completion?